centaur/src/main/resources/standardTestCases/drs_usa_hca_data_in_jdr.test

```diff
@@ -0,0 +1,25 @@
+name: drs_usa_hca_data_in_jdr
+testFormat: WorkflowSuccess
+backends: ["papi-v2-usa"]
+skipDescribeEndpointValidation: true
+
+files {
+  workflow: drs_tests/drs_usa_hca_data_in_jdr.wdl
+  options-dir: "Error: BA-6546 The environment variable CROMWELL_BUILD_RESOURCES_DIRECTORY must be set/export pointing to a valid path such as '${YOUR_CROMWELL_DIR}/target/ci/resources'"
+  options-dir: ${?CROMWELL_BUILD_RESOURCES_DIRECTORY}
+  options: ${files.options-dir}/papi_v2_usa_compute_default.options.json
```

Comment:
```suggestion
  options: ${files.options-dir}/papi_v2_usa.options.json
```

That suggested json uses "normal" dev service accounts. The `compute_default` GCE SA is not a member of "[All Terra users](https://bvdp-saturn-dev.appspot.com/#workspaces/general-dev-billing-account/DRS%20Test%20Workspace%20-%20Dev/data)". That old JSON can be deleted.